packages/neo4j-driver/test/types/driver.test.ts

```diff
@@ -135,6 +135,15 @@ const rxSession9: RxSession = driver.rxSession({
   bookmarks: ['bookmark1', 'bookmark2']
 })
 
+const rxSession10: RxSession = driver.rxSession({
```

Comment:
where are those used?

Reply:
The type tests are a bit strange, we just call a method with the expected params and receiving the expect results. Then we check if it get perfect checked by the TS compiler.

Reply:
or it just to exercise the typescript compiler?

Reply:
Yes, to check if it get typed correctly